Chat/Chat.csproj

```diff
@@ -9,6 +9,7 @@
     <DefaultItemExcludes>$(DefaultItemExcludes);$(SpaRoot)node_modules\**</DefaultItemExcludes>
   </PropertyGroup>
 
+  <PropertyGroup Condition=" '$(RunConfiguration)' == 'Web' " />
```

Comment:
Do we need to checkin those changes?

Reply:
Ah this a good point. I think these changes were made when I just opened in visual studio